inspirehep/utils/url.py

```diff
@@ -65,9 +65,9 @@ def is_pdf_link(url):
     except requests.exceptions.RequestException:
         return False
 
-    significant_bytes = next(response.iter_content(10000), '').strip()
+    found = next(response.iter_content(10000), '').find('%PDF')
```

Comment:
why rename it from ``significant_bytes`` and not do the test speratly? It's a bit confusing IMO :/

Reply:
Because this are not significant bytes anymore? This variable name was confusing.

Reply:
I see that this has been done like that before, but I do not get why we just don't check `Content-Type: application/pdf` response header :/

Reply:
Looks like this is a requirement not to check it: https://github.com/inspirehep/inspire-next/blob/28630eab052ef798a874f9ca4a731c5ffeab9782/tests/unit/forms/test_forms_validators_simple_fields.py#L139

Reply:
you absolutely don't want to rely on `Content-Type` for file format verification. Most sites (web servers) use simple mime type based on file extension to set `Content-Type` for static files. It doesn't guarantee that the file actually is what it claims. A proper check is to do introspection,
at least use file magic (there's pleny of libraries for that) or if you want to be more thorough use JHOVE https://jhove.openpreservation.org/  or similar